app/models/pacbio/well.rb

```diff
@@ -52,6 +57,46 @@ class Well < ApplicationRecord
 
     delegate :run, to: :plate, allow_nil: true
 
+    def pool_ids=(ids)
+      ids.each do |id|
+        # If the used_aliquot already exists, skip it
+        next if used_aliquots.find_by(source_id: id)
+
+        # Create a new used_aliquot if it doesn't exist
+        used_aliquots << used_aliquots.build(source_id: id, source_type: 'Pacbio::Pool',
+                                             volume: 0, concentration: 0, aliquot_type: :derived,
```

Comment:
I have given these a default for the time being since they need to pass validation. We can skip validation, not sure if thats something we want to be doing though.

Reply:
I think this is an acceptable way to do this. When we come to do volume tracking we will need it.